pandas/util/testing.py

```diff
@@ -1652,6 +1652,87 @@ def makeMultiIndex(k=10, names=None, **kwargs):
     return MultiIndex.from_product((("foo", "bar"), (1, 2)), names=names, **kwargs)
 
 
+_names = [
+    "Alice",
+    "Bob",
+    "Charlie",
+    "Dan",
+    "Edith",
+    "Frank",
+    "George",
+    "Hannah",
+    "Ingrid",
+    "Jerry",
+    "Kevin",
+    "Laura",
+    "Michael",
+    "Norbert",
+    "Oliver",
+    "Patricia",
+    "Quinn",
+    "Ray",
+    "Sarah",
+    "Tim",
+    "Ursula",
+    "Victor",
+    "Wendy",
+    "Xavier",
+    "Yvonne",
+    "Zelda",
+]
+
+
+def make_timeseries(start="2000-01-01", end="2000-12-31", freq="1D", seed=None):
```

Comment:
i know this is how dask names things but this is confusing here as it returns a frame

also is it possible to make this as a wrapper around makeTimessries and/or getTimeSeriesData (which, which should rename to python case, OT)